app/templates/administrative-units/administrative-unit/ministers/new.hbs

```diff
@@ -5,6 +5,30 @@
   <PageHeader class="au-u-margin au-u-margin-bottom-large">
     <:title>Voeg bedienaar toe</:title>
     <:subtitle>{{@model.administrativeUnit.name}}</:subtitle>
+
+    <:action>
+      {{#if this.isSelectingTargetPerson}}
```

Comment:
This check needs to be reversed I think? At the moment the buttons are still visible in the "person selection" stage but not in the form.

Reply:
ah yes sorry, stupid mistake. thought it was an easy one so went too fast, next time i'll be more careful, sorry for  that

Reply:
when I committed  it changed from "if (not" to "unless"

Reply:
Yea, that's because of the [no-negated-condition](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-negated-condition.md) linting rule which has an autofixer (which runs precommit because of husky).

I'm not a big fan of that rule, I think `if not something` is way more readable than `unless something`. But it's part of the default ruleset and we try to not overrule those as much as possible. I hope this default rule changes in the future though and there are more people who have a similar mindset: https://github.com/emberjs/rfcs/issues/714